Pinta.Effects/Effects/GradientHelper.cs

```diff
@@ -1,9 +1,23 @@
+using System;
 using System.Collections.Generic;
+using System.Linq;
 using Pinta.Core;
 using Pinta.Gui.Widgets;
 
 namespace Pinta.Effects;
 
+public enum ColorSchemeSource
+{
+	[Caption ("Predefined Gradient")]
```

Comment:
I'd prefer `Preset Gradient`  as the label for the user here

Reply:
No problem. The label has been changed